WcaOnRails/spec/models/user_spec.rb

```diff
@@ -510,4 +510,18 @@
       end
     end
   end
+
+  describe "#notify_of_results_presence" do
+    let(:competition) { FactoryGirl.build(:competition) }
+
+    it "sends the notification if the user has it enabled" do
+      expect(CompetitionsMailer).to receive(:notify_users_of_results_presence).and_call_original
```

Comment:
With these tests, I believe you could be little stricter and specify `.with(...)` to expect the correct arguments being passed to the `notify_users_of_results_presence` method. For example, see here: https://github.com/jfly/worldcubeassociation.org/blob/59479df9052ee6b567f556e70f5ed90cd21b00f8/WcaOnRails/spec/controllers/registrations_controller_spec.rb#L65